### DISCUSSION TITLE
Integration with dlisio

### DISCUSSION THREAD

**Discussion by fbordignon** [well-id/dliswriter#35] [General]
Hello guys, great work with the lib.
dliswriter is exactly what we needed!
Before finding out about dliswriter, we were thinking of contributing to dlisio to add writing functionalities, since it is a well maintained lib and backed by Equinor.
Do you guys have specific reasons to create a new lib and not contribute to dlisio?
Thanks!

**Comment by the-mysh** [comment]
Hi, great to hear it's coming in useful!
The decision to keep it separate was made a few years ago, before my time at Well ID, so I'm not sure about the exact initial reasons. In any case, we'd be open to turn it into a contribution to dlisio if dlisio maintainers are interested in that!
